### PR TITLE
Linux Install Update and Github Formatting

### PR DESCRIPTION
**Pull request by giantcow**
- Changed linux install guild to use n with n installer.
- Updated files in ./.github/ to meet markdownlint standards.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
+
 Please follow these rules when making contributions to this repository.
 
 ## General contribution rules
+
 1. All pull requests go to the review branch.
-2. You must have good written English skills, pulls with grammar errors will get change requests.
-3. Follow the overall format in the docs.
-4. Leave image embedding to the official writers.
+1. You must have good written English skills, pulls with grammar errors will get change requests.
```
Comment (Dougley) [inline .github/CONTRIBUTING.md]: If you're going to ditch the numbers, make them dot notation instead.

File: docs/install_docker.md [hunk 2]
```diff
@@ -124,6 +124,9 @@ imgflip | Your [imgflip](https://imgflip.com) login details. | Used in certain m
 google | For retrieving data from YouTube. | Refer to the "Editing the settings" section.
 mashape | For retrieving the Fortune Cow. | Refer to the "Editing the settings" section.
 twitchId | For retrieving status of whether a user on Twitch is streaming or not. | Refer to the "Editing the settings" section.
+imgur | For retrieving random memes from Imgur. | Refer to the "Editing the settings" section.
+cleverbot_user | Username to the cleverbot.io API, used in the cleverbot command. | Refer to the "Editing the settings" section.
+cleverbot_key | Key to the cleverbot.io API, used in the cleverbot command. | Refer to the "Editing the settings" section.
```
Comment (Dougley) [inline docs/install_docker.md:129]: Have we updated the Dockerfile to support this yet?

**Review by Dougley — CHANGES_REQUESTED** [review]
I know I don't maintain the docs, but I can nitpick a bit.

File: docs/install_docker.md (hunk 2 above)
Comment (linuswillner) [reply docs/install_docker.md:129]: https://github.com/TheSharks/WildBeast/blob/master/Dockerfile#L19

File: docs/install_linux.md [hunk 3]
```diff
@@ -22,20 +22,22 @@ You will be asked for a password when doing the command, that's all that really
...
-sudo apt-get install -y curl
-curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
-sudo apt-get install -y nodejs
-sudo apt-get install -y build-essential
+# Install prerequisites
+sudo apt-get install -y curl make build-essential
+# Install n and node lts
+curl -L https://git.io/n-install | bash -s -- -y lts
+# Reload bash
+source ~/.bashrc	# ZSH users source their ~/.zshrc file
```
Comment (linuswillner) [inline docs/install_linux.md:32]: Has all this been tested to be functional? (Lines 27-32)

File: docs/install_linux.md [hunk 4]
```diff
@@ -22,20 +22,22 @@ You will be asked for a password when doing the command, that's all that really
 **Keep this in mind when setting up, so you don't waste our time with errors that stem from lacking permissions!**
 While you may think running the following commands as the root user is easy, we advice against it due to the security risks involved when using such a powerful user, instead please use `sudo`.
 ##Installing Node
-First off, we'll install Node.JS, the runtime WildBeast uses. The current 6.x.x version works fine.
+First off, we'll install Node.JS, the runtime WildBeast uses, with [n-install](https://github.com/mklement0/n-install) which installs [n](https://github.com/tj/n). ***Note:*** n installs node *per user* so if you have more than one user on your machine, they must also install n.
```
Comment (linuswillner) [inline docs/install_linux.md]: Use bold font for emphasis, not bold and cursive at the same time.

**Review by linuswillner — CHANGES_REQUESTED** [review]
2 changes and questions about this. Otherwise I'm okay with the changes.

File: docs/install_linux.md (hunk 3 above)
Comment (giantcow) [reply docs/install_linux.md:32]: Yes

File: docs/install_linux.md (hunk 4 above)
Comment (giantcow) [reply docs/install_linux.md]: Done in [8dfefbb](https://github.com/TheSharks/WildBeastDocs/pull/20/commits/8dfefbb0771518ab218dfcdd5dbe6db8a8f57b94)

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (giantcow) [reply .github/CONTRIBUTING.md]: Done in [d8ecdec](https://github.com/TheSharks/WildBeastDocs/pull/20/commits/d8ecdecfe1bdba998bb04c9c2d8a9566c1399b8c)

**Review by Dougley — APPROVED** [review]

File: docs/install_linux.md (hunk 3 above)
Comment (linuswillner) [reply docs/install_linux.md:32]: Okay, cool, just wanted to know. Cheers 👍

**Review by linuswillner — APPROVED** [review]

**Pull request merged**